src/cosmosdb-preview/azext_cosmosdb_preview/tests/latest/test_cosmosdb-pitr_scenario.py

```diff
@@ -820,3 +820,64 @@ def test_cosmosdb_system_identity_restore(self, resource_group):
 
         restored_account_defaultIdentity = restored_account['defaultIdentity']
         assert user_id_2 in restored_account_defaultIdentity
+
+        public_network_access = restored_account['publicNetworkAccess']
+        assert public_network_access == 'Disabled'
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(name_prefix='cli_test_cosmosdb_cross_region_restore', location='eastus2')
+    def test_cosmosdb_xrr(self, resource_group):
+        col = self.create_random_name(prefix='cli', length=15)
+        db_name = self.create_random_name(prefix='cli', length=15)
+        source_acc = self.create_random_name(prefix='cli-continuous30-', length=25)
+        target_acc = source_acc + "-restored"
+        loc = 'eastus2'
+        target_loc = 'centralus'

```

Comment:
we should rename or explain what this mean.  Looks like this is being used as a read region in line 846, so we need to explain why it is called "target_loc"

Reply:
added a comment there